Makefile

```diff
@@ -18,24 +18,22 @@ generate:
 	cp plugins.json data/static/
 	go generate ./...
 
-## Runs govet and gofmt against all packages.
+## Runs go vet and golangci-lint against all packages.
 .PHONY: check-style
-check-style: govet lint
-
-## Runs govet against all packages.
-.PHONY: govet
-govet:
+check-style:
 	go vet ./...
 
-## Runs lint against all packages.
-.PHONY: lint
-lint:
-	GO111MODULE=off go get -u golang.org/x/lint/golint
-	golint -set_exit_status ./...
+# https://stackoverflow.com/a/677212/1027058 (check if a command exists or not)
+	@if ! [ -x "$$(command -v golangci-lint)" ]; then \
+		echo "golangci-lint is not installed. Please see https://github.com/golangci/golangci-lint#install for installation instructions."; \
+		exit 1; \
+	fi; \
+
+	golangci-lint run --max-same-issues=10 ./...
 
 ## Runs test against all packages.
 .PHONY: test
-test:
+test: check-style
```

Comment:
In all other projects we don't run style checks alongside with tests. My concern is, that the style checks might take some seconds and prevent devs from quickly running the tests. Also, I think it's fine to run tests without passing style checks.

Reply:
Seems reasonable: will break it back out.